unit_tests/ut_h_stats.amlg

```diff
@@ -388,6 +388,68 @@
 		percent 0.45
 	))
 
+	(assign (assoc
+		result
+			(call_entity "howso" "react_aggregate" (assoc
+				action_feature "fruit"
+				details (assoc
+					feature_contributions_full (true)
+					feature_mda_full (true)
+					prediction_stats (true)
+					selected_prediction_stats (list "accuracy" "precision" "recall" "mae" "spearman_coeff" "mcc" "rmse")
+				)
+			))
+	))
+
+	(call keep_result_payload)
+
+	(declare (assoc
+		result2
+			(get
+				(call_entity "howso" "react_aggregate" (assoc
+					action_feature "fruit"
+					details (assoc
+						feature_contributions_full (true)
+						feature_mda_full (true)
+						prediction_stats (true)
+						selected_prediction_stats (list "accuracy" "precision" "recall" "mae" "spearman_coeff" "mcc" "rmse")
+					)
+				))
+				(list 1 "payload")
+			)
+	))
+
+	;for react_aggregate, 'always_recalculate' is turned off by default
+	(print "Output the same stats when react_aggregate is called twice, thus retrieving the cached values")
+	; (call assert_same (assoc
```

Comment:
surely these asserts should be getting from result and result2?

Reply:
yes, forgot to change back